xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -107,3 +114,104 @@ async def send_reliable_submission(
     return await _wait_for_final_transaction_outcome(
         transaction_hash, client, prelim_result, transaction.last_ledger_sequence
     )
+
+
+def _decode_tx_blob(tx_blob: str) -> Transaction:
+    """
+    Decodes a transaction blob.
+
+    Args:
+        tx_blob: the tx blob to decode.
+
+    Returns:
+        The formatted transaction.
+    """
+    return Transaction.from_xrpl(decode(tx_blob))
+
+
+async def _get_signed_tx(
+    transaction: Union[Transaction, str],
+    client: Client,
+    wallet: Optional[Wallet] = None,
+    check_fee: bool = True,
+    autofill: bool = True,
+) -> Transaction:
+    """
+    Initializes a signed transaction for a submit request.
+
+    Args:
+        transaction: the transaction or transaction blob to be signed if unsigned.
+        client: the network client to autofill from.
+        wallet: the wallet with which to sign the transaction (optional, only needed
+        if the transaction is not signed).
+        check_fee: an optional bolean indicating whether to check if the fee is
+            higher than the expected transaction type fee. Defaults to True.
+        autofill: an optional boolean indicating whether to autofill the
+            transaction. Defaults to True.
+
+    Returns:
+        The signed transaction.
+    """
+    if isinstance(transaction, str):
+        transaction = _decode_tx_blob(transaction)
+
+    if transaction.is_signed():
+        return transaction
+
+    if not wallet:
+        raise XRPLException(
+            "Wallet must be provided when submitting an unsigned transaction"
+        )
+
+    if check_fee:
+        await _check_fee(transaction, client)
+
+    if autofill:
+        transaction = await _autofill(transaction, client)
+
+    return (
+        await sign(transaction, wallet, True)
+        if transaction.signers
+        else await sign(transaction, wallet, True)
+    )
+
+
+async def submit_and_wait(
+    transaction: Union[Transaction, str],
+    client: Client,
+    wallet: Optional[Wallet] = None,
+    *,
+    check_fee: bool = True,
+    autofill: bool = True,
+    fail_hard: bool = False,
+) -> Response:
+    """
+    Signs a transaction locally, without trusting external rippled nodes (only if
+    the input transaction is unsigned; otherwise, proceeds to the next steps), submits,
+    and verifies that it has been included in a validated ledger (or has errored
+    /will not be included for some reason).
+    `See Reliable Transaction Submission
+    <https://xrpl.org/reliable-transaction-submission.html>`_
```

Comment:
* Adding context for why they may want to look up that bigger reference
* The parentheses parts are implied by the "if ..." so can be cut
* Moved the "/" to the top so it seems like punctuation, rather than part of the next word (like a /command)
```suggestion
    Signs a transaction locally if the transaction is unsigned, then submits,
    and verifies that it has been included in a validated ledger (or has errored/
    will not be included for some reason).
    `See Reliable Transaction Submission for a full explanation of this workflow
    <https://xrpl.org/reliable-transaction-submission.html>`_
```